### ISSUE TITLE
front-end error returned upon logout

### ISSUE THREAD

**Issue by ericpaulsen** [coder/coder#10494]
a customer reported this error when they logged out of Coder:

```
Error: User is not authenticated.
    at Ni (https://<url>/assets/index-c14b02b0.js:329:10969)
    at iCe (https://<url>/assets/index-c14b02b0.js:625:6552)
    at a2 (https://<url>/assets/index-c14b02b0.js:61:19538)
    at T3 (https://<url>/assets/index-c14b02b0.js:63:44055)
    at E3 (https://<url>/assets/index-c14b02b0.js:63:39787)
    at k7 (https://<url>/assets/index-c14b02b0.js:63:39715)
    at oh (https://<url>/assets/index-c14b02b0.js:63:39568)
    at U0 (https://<url>/assets/index-c14b02b0.js:63:35933)
    at wE (https://<url>/assets/index-c14b02b0.js:63:36737)
    at aa (https://<url>/assets/index-c14b02b0.js:61:3279)
```

**Comment by BrunoQuaresma** [comment]
I think we had a similar issue a few weeks ago, what is the customer version?

**Comment by ericpaulsen** [comment]
customer is on 2.3.1 - this appears to be resolved in 2.3.3 via #10349
